molecule/shared/substrate/lxd/destroy.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Destroy LXD containers
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  vars:
+    molecule_yml:
+      platforms:
+        - name: instance
+          image: ubuntu:20.04
+          user: ansible
```

Comment:
Should be pulled in like in the `create.yml` file. This should be removed. If the image format doesn't work we should fix in the `molecule.yml` file not overriding it here.